Jenkinsfile

```
@@ -77,8 +78,10 @@ def setupNodeAndTest(version, filter='', testSuite='test') {
                 npm install mocha-jenkins-reporter --save-dev
                 curl -O -u "\${ARTIFACTORY_USER}:\${ARTIFACTORY_PW}" "https://na.artifactory.swg-devops.com/artifactory/cloudant-sdks-maven-local/com/ibm/cloudant/${env.DBCOMPARE_NAME}/${env.DBCOMPARE_VERSION}/${env.DBCOMPARE_NAME}-${env.DBCOMPARE_VERSION}.zip"
                 unzip ${env.DBCOMPARE_NAME}-${env.DBCOMPARE_VERSION}.zip
-                export COUCH_BACKEND_URL="https://\${DB_USER}:\${DB_PASSWORD}@\${SDKS_TEST_SERVER_HOST}"
+                set +x
+                export COUCH_BACKEND_URL="https://\${DB_USER}:${dbPassword}@\${SDKS_TEST_SERVER_HOST}"
```

Have you tried to use: https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#interpolation-of-sensitive-environment-variables?
```suggestion
                export COUCH_BACKEND_URL='https://\${DB_USER}:${dbPassword}@\${SDKS_TEST_SERVER_HOST}'
```
Based on my local test, this will result:
`export COUCH_BACKEND_URL=https://${DB_USER}:${DB_PASSWORD}@${SDKS_TEST_SERVER_HOST}`

---

`sh'''` syntax is used during git config in sdks.

---

no, this doesn't work on two levels: 1) this doesn't mask password 2) it does direct interpolation of vars, so we are not getting _value_ of `${DB_USER}`, but literal name of variable, e.g. it looks like `COUCH_BACKEND_URL=https://${DB_USER}:xxxx@${SDKS_TEST_SERVER_HOST}` which obviously doesn't work